src/main/java/com/netflix/imflibrary/st2067_2/IMFCompositionPlaylistType.java

```diff
@@ -440,8 +457,21 @@ public String getCoreConstraintsVersion() {
      *
      * @return a string representing ApplicationIdentification for this CompositionPlaylist
      */
+    @Deprecated
     public String getApplicationIdentification() {
-        return this.applicationId;
+        if (this.applicationIdSet.size() > 0) {
+            return this.applicationIdSet.iterator().next();
+        } else {
+            return null;
```

Comment:
The old applicationId was being initialized as an empty string, rather than a null.
The change for compatibility should maybe use `return "";` instead.

Reply:
Good catch. Will fix.